oci/airflow-rocks/documentation.yaml

```diff
@@ -0,0 +1,144 @@
+  - type: -e
+    value: 'AIRFLOW__CELERY__RESULT_BACKEND=db+postgresql://airflow:airflow@db:5432/airflow'
+    description: >
+      Task result backend for CeleryExecutor. Not used in standalone mode.
+  - type: -e
+    value: 'AIRFLOW_UID=50000'
+    description: >
+      Useful if you want to control which UID writes files under mounted volumes,
+      mirroring upstream Airflow container practice.
+  - type: -e
+    value: 'AIRFLOW_GID=50000'
+    description: >
+      Group ID to match host perms for mounted volumes.
+  - type: -p
+    value: '8080:8080'
+    description: >
+      Airflow Webserver UI (HTTP). This is what you open in the browser.
+
+  - type: -p
+    value: '8793:8793'
+    description: >
+      Airflow triggerer / debug / health endpoints can run here in some setups;
+      include if you split services or run triggerer separately.
+      (Not strictly needed for `airflow standalone`, but common in prod charts.)
+  - type: -v
+    value: '/path/to/dags:/var/lib/airflow/dags'
+    description: >
+      Mount your DAGs from the host so you can iterate without rebuilding the rock.
+      Anything in /var/lib/airflow/dags will be auto-discovered.
+
+  - type: -v
+    value: '/path/to/logs:/var/lib/airflow/logs'
+    description: >
+      Persist task logs outside the container (useful for debugging and restarts).
+
+  - type: -v
+    value: '/path/to/airflow.cfg:/var/lib/airflow/airflow.cfg'
+    description: >
+      Inject a custom airflow.cfg instead of the auto-generated one from
+      `airflow standalone`.
+
+  - type: -v
+    value: '/path/to/requirements.txt:/requirements.txt'
+    description: >
+      (Optional) If you extend this rock and want extra Python deps at runtime,
+      you can mount and install them in an init hook / derived image.
+  - type: -e
+    value: 'AIRFLOW__SCHEDULER__MAX_THREADS=4'
+    description: >
+      Parallelism of the scheduler loop. Increase for heavier DAG throughput.
+  - type: -e
+    value: 'AIRFLOW__CORE__PARALLELISM=32'
+    description: >
+      Global max task instances that can run in parallel across the whole cluster.
+  - type: -e
+    value: 'AIRFLOW__CORE__DAG_CONCURRENCY=16'
+    description: >
+      Max number of task instances allowed to run per DAG.
+  - type: -e
+    value: 'AIRFLOW__WEBSERVER__WORKERS=4'
+    description: >
+      Gunicorn worker count for the web UI.
+  - type: -e
+    value: 'AIRFLOW__API__AUTH_BACKENDS=airflow.api.auth.backend.basic_auth'
+    description: >
+      Enable basic auth for the stable API. For production you'd point to a more
+      secure auth backend.
+  - type: -e
+    value: 'AIRFLOW__WEBSERVER__SECRET_KEY=change-me'
+    description: >
+      Flask secret key used by the webserver for session signing. MUST be set
+      to a strong value in production.
+  - type: -e
+    value: 'AIRFLOW__WEBSERVER__BASE_URL=https://airflow.example.com'
+    description: >
+      External URL for the web UI. Helps generate links in emails, etc.
```

Comment:
I believe this is now called AIRFLOW__API__BASE_URL, which belongs to the [api] configuration. Most of the webserver configurations were added to [api] after 3.x. Please confirm, though.